cheroot/ssl/builtin.py

```diff
@@ -102,12 +136,42 @@ def __init__(
         if self.ciphers is not None:
             self.context.set_ciphers(ciphers)
 
+        self._server_env = self.env_cert_dict(
+            'SSL_SERVER', self.parse_cert(certificate, private_key),
+        )
+        if self._server_env:
+            cert = None
+            with open(certificate, mode='rt') as f:
+                cert = f.read()
+
+            # strip off any keys by only taking the first certificate
+            if all(x in cert for x in {ssl.PEM_HEADER, ssl.PEM_FOOTER}):
+                cert = cert.split(ssl.PEM_HEADER, 1)[-1]
```

Comment:
Are you trying to do just:
```suggestion
                cert_start = cert.find(ssl.PEM_HEADER)
                cert_end = cert.find(ssl.PEM_FOOTER) + len(ssl.PEM_FOOTER)
                cert = cert[cert_start:cert_end]
```
Splitting and recreating reads cryptic...

Reply:
updated to use find

Reply:
FTR it's a good tone to accept suggested changes via the GitHub UI button because it automatically credits the co-author.

Reply:
sorry, I wasn't aware of that. thanks for letting me know